app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -1029,19 +1030,20 @@ export default class SampleViewV2 extends React.Component {
               />
             </div>
           )}
-          {view == "tree" && (
-            <div>
-              <TaxonTreeVis
-                lineage={lineageData}
-                metric={selectedOptions.metric}
-                nameType={selectedOptions.nameType}
-                onTaxonClick={this.handleTaxonClick}
-                sample={sample}
-                taxa={filteredReportData}
-                useReportV2Format={true}
-              />
-            </div>
-          )}
+          {view == "tree" &&
+            filteredReportData.length > 0 && (
```

Comment:
did we consider adding an empty state message to the user such as "no data passed the selected filters" ?

Reply:
There's a small line underneath the filters that tells the user how many rows are passing the selected filters, which will say "0 rows passing the above filters, out of # total rows."